src/gui/guiFormSpecMenu.cpp

```diff
@@ -4060,6 +4062,34 @@ bool GUIFormSpecMenu::OnEvent(const SEvent& event)
 {
 	if (event.EventType==EET_KEY_INPUT_EVENT) {
 		KeyPress kp(event.KeyInput);
+		if (event.KeyInput.PressedDown) {
```

Comment:
Please explain what this code should do:
```suggestion
		// Ctrl (+ Shift) + Tab: Select the (previous or) next tab of a TabControl instance.
		if (event.KeyInput.PressedDown) {
```